<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmTime</class>
 <widget class="QWidget" name="FrmTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>404</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblHH">
      <property name="text">
       <string>HH:</string>
      </property>
      <property name="buddy">
       <cstring>editHH</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editHH">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Enter hours from 0 to 24</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lblMM">
      <property name="text">
       <string>MM:</string>
      </property>
      <property name="buddy">
       <cstring>editMM</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="editMM">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Enter minutes from 0 to 60</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="lblSS">
      <property name="text">
       <string>SS:</string>
      </property>
      <property name="buddy">
       <cstring>editSS</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="editSS">
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Enter seconds and nano seconds using format nn.nnnnnnnnn</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="lblTZ">
      <property name="text">
       <string>TZ:</string>
      </property>
      <property name="buddy">
       <cstring>editSS</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="editTZ">
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Enter timezone using format +/-hhmm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
